python/pyspark/sql/session.py

```diff
@@ -1251,7 +1252,7 @@ def _getActiveSessionOrCreate(**static_conf: Any) -> "SparkSession":
             spark = builder.getOrCreate()
         return spark
 
-    @overload
+    @overload  # type: ignore[override]
```

Comment:
PyArrow's lack of type annotations makes this ignore necessary. I tried a variety of different ways to avoid adding it, with no success.

Reply:
Are you sure this isn't due to the fact that the provided overload doesn't have the `schema` argument? Overriding a method while adding or removing arguments will generally trigger an error in mypy (which override does mute)

Reply:
Yes, I didn't change anything involving the `schema` argument. I think the problem is that PyArrow classes appear in mypy type check results as `Any`, as described at https://github.com/apache/arrow/issues/24376#issuecomment-1377869384. The type checker doesn't like this.